.github/workflows/welcome.yml

```
@@ -0,0 +1,18 @@
+name: Welcome New Contributor
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: |-
+            Congrats on making your first Pull Request to Open Sauced! 🎉🍕
+            Join the conversation in our [Discord](https://discord.gg/gZMKK5q)
+            Thanks for taking the time to improve Open Sauced! ❤️
```

The call to action should be clearer. 

```suggestion
            Congrats on making your first Pull Request and thanks for taking the time to improve Open Sauced! ❤️! 🎉🍕
            Say hello by joining the conversation in our [Discord](https://discord.gg/gZMKK5q)
```